src/mono/mono/metadata/image-internals.h

```diff
@@ -9,6 +9,17 @@
 #include <mono/metadata/image.h>
 #include <mono/metadata/loader-internals.h>
 
+typedef struct {
+	int dont_care_about_cli : 1;
```

Comment:
Any reason why we inverted these parameters compared to the arguments passed to functions below?

Reply:
Any particular win to use bit fields for this? I think it would be much clearer to use bool type instead, it would also make the init more clear.

Reply:
> Any specific reason why these parameters where inverted compared to the arguments passed to functions below?

Turns out most of the time we care about cli and pecoff, so the callers become a bit simpler: they can just initialize the options struct to zero.

> Any particular win to use bit fields for this? I think it would be much clearer to use bool type instead, it would also make the init code more clear.

I think I suggested bitfields more out of habit than any rational reason. These could be booleans.

Reply:
Thanks! I'll modify these to a boolean type in the upcoming PR

Reply:
This is now in PR #71657 @lateralusX @lambdageek